src/main/java/com/github/packageurl/PackageURL.java

```diff
@@ -283,17 +294,23 @@ private static boolean isValidCharForKey(int c) {
         return (isAlphaNumeric(c) || c == '.' || c == '_' || c == '-');
     }
 
-    private static void validateChars(String value, IntPredicate predicate, String component) throws MalformedPackageURLException {
+    private static void validateChars(String value, IntPredicate predicate, String component)
+            throws MalformedPackageURLException {
         char firstChar = value.charAt(0);
 
         if (isDigit(firstChar)) {
-            throw new MalformedPackageURLException("The PackageURL " + component + " cannot start with a number: " + firstChar);
+            throw new MalformedPackageURLException(
+                    "The PackageURL " + component + " cannot start with a number: " + firstChar);
         }
 
-        String invalidChars = value.chars().filter(predicate.negate()).mapToObj(c -> String.valueOf((char) c)).collect(Collectors.joining(", "));
+        String invalidChars = value.chars()
+                .filter(predicate.negate())
+                .mapToObj(c -> String.valueOf((char) c))
+                .collect(Collectors.joining(", "));
 
         if (!invalidChars.isEmpty()) {
-            throw new MalformedPackageURLException("The PackageURL " + component + " '" + value + "' contains invalid characters: " + invalidChars);
+            throw new MalformedPackageURLException(
+                    "The PackageURL " + component + " '" + value + "' contains invalid characters: " + invalidChars);
```

Comment:
Maybe increase the line length? What is it currently?

Reply:
The line length chosen by the `palantir-java-format` is 120. Non-modifiable.

In general I like the fact that the formatter does not accept any parameters. At a previous job we spent 3 months deciding on all the hundreds of options of the Eclipse formatter and everybody hated the result anyway.

Reply:
Actually, 120 is OK (it has to be _something_ I guess.